tests/integration/criteria/criterion-list-entry.test.ts

```diff
@@ -0,0 +1,68 @@
+import CriterionListEntry from "$lib/components/composites/criteria/CriterionListEntry.svelte";
+import { render, screen } from "@testing-library/svelte";
+import { describe, expect, test } from "vitest";
+import { createReviewedCriterion } from "../../model-builder";
+import { Reviews, Users } from "../../example-data";
+
+describe("CriterionListEntry", () => {
+    test("When props are provided, then component is shown", () => {
+        render(CriterionListEntry, {
+            target: document.body,
+            props: {
+                inReviewMode: true,
+                reviewers: [],
+                criterion: createReviewedCriterion({
+                    tag: "C1",
+                    name: "Example Criterion",
+                }),
+            },
+        });
+
+        const tag = screen.getByText("C1");
+        const name = screen.getByText("Example Criterion");
+
+        expect(tag).toBeInTheDocument();
+        expect(name).toBeInTheDocument();
+    });
+
+    test("When `isReviewMode` is true, then the checkbox is shown", () => {
```

Comment:
```suggestion
    test("When `inReviewMode` is true, then the checkbox but no user avatars are shown", () => {
```

In gerenal, I like the idea to test both things in one test, but even if the reviewMode would be true, your last test case would pass, as you dont add any reviewers that could be shown, but aren't due to the `inReviewMode` flag.

Reply:
True that. I added them.